Clean up gradle files and dependencies and remove unnecessary code.

hemikak opened the pull request:
## Purpose
> Removed unnecessary code files. Cleaned up gradle files and updated the existing dependencies.

hemikak commented on build.gradle:
```diff
@@ -335,9 +262,4 @@ task codeCoverageReport(type: JacocoReport) {
-}
-//buildPlugin.dependsOn check
\ No newline at end of file
+}
```
Fixed in https://github.com/ballerina-platform/ballerina-openapi/pull/52/commits/7bf4114363461fb30c977a875efc5571b6d7a929

hemikak commented on build.gradle:
```diff
@@ -172,13 +158,6 @@ allprojects {
                 password System.getenv("packagePAT")
             }
         }
-//        maven {
```
Fixed in https://github.com/ballerina-platform/ballerina-openapi/pull/52/commits/7bf4114363461fb30c977a875efc5571b6d7a929

hevayo reviewed (commented):
Have we removed @openapi:ClientEndpoint annotation which is no longer relevant

hemikak commented:
> Have we removed @openapi:ClientEndpoint annotation which is no longer relevant

Removed in https://github.com/ballerina-platform/ballerina-openapi/pull/52/commits/69dd3a0b63e417a45d9af3614d8f920e52fb101e
